docs/pages/guides/using-vexo.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Using Vexo
+maxHeadingDepth: 4
+hideTOC: true
+description: A guide on installing and configuring Vexo for real-time user analytics.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+[Vexo](https://www.vexo.co/) provides real-time user analytics for your Expo application, helping you understand how users interact with your app, identify friction points, and improve engagement.
+
+With a two-line integration, Vexo starts collecting data automatically, giving you actionable insights to optimize your app's user experience. If needed, you can also create custom events.
+
+## Features
+
+1. **Complete Dashboard**
+   - Active Users
+   - Session Time
+   - Downloads
+   - OS Distribution
+   - Version Adoption
+   - Geographic Insights
+   - Popular Screens
+2. **Session Replays**
+   - Watch real user sessions to understand their interactions.
+3. **Heatmaps**
+   - Identify the most engaged areas of your app.
+4. **Funnels**
+   - Analyze user flows and optimize conversion rates.
+5. **Custom Events and Dashboard Personalization**
+   - Track specific user actions by creating custom events.
+   - Customize your dashboard to visualize key metrics.
+
+## Getting started
+
+1. Create an account: Sign up for a [Vexo account](https://www.vexo.co/).
+2. Create a new app: You'll be prompted to create a new app. Give it a name (you can change it later), and once you submit it, you'll receive an API key.
+3. Install the Vexo package: Run the following command in your project:
+
+   <Tabs>
+     <Tab label="npm">
+       <Terminal cmd={['$ npm install vexo-analytics']} />
+     </Tab>
+     <Tab label="yarn">
+       <Terminal cmd={['$ yarn add vexo-analytics']} />
+     </Tab>
+   </Tabs>
+
+4. Initialize Vexo: Add the following code in your app's entry file (for example, **index.js**,
+   **App.js**, or **\_layout.tsx** if using Expo Router):
```

Comment:
```suggestion
   **App.js**, or **app/\_layout.tsx** if using Expo Router):
```

Let's mention the `app/` prefix to indicate that it has to be used in the Root layout file.